src/server/stats/stats_mixin.js

```diff
@@ -0,0 +1,19 @@
+import { getStats } from './stats';
+
+export function statsMixin(kbnServer, server) {
+  server.route({
+    method: 'GET',
+    path: '/api/stats',
+    handler: async function (request, reply) {
+      const { callWithInternalUser } = server.plugins.elasticsearch.getCluster('admin');
+      const callAdminCluster = (...args) => callWithInternalUser(...args);
+
+      const stats = await getStats(
+        server.config().get('kibana.index'),
+        callAdminCluster
+      );
+
+      return reply(stats);
```

Comment:
It worked when I tested it, but using the above doesn't seem to work (I don't see an error, it just hangs).  Do I need to wrap your return with the reply?